jenkins/ci.suse.de/openstack-ardana-pcloud.yaml

```diff
@@ -104,6 +104,11 @@
           description: >-
             The Jenkins agent where this job must run. Used by upstream jobs to
             force a job to reuse the same node.
+      - string:
+          name: external_name
+          default: ''
+          description: >-
+            Configurable external name for your public endpoints
 
```

Comment:
Don't we need to test this in virtual environments too?  So, it may be better to expose this parameter in the upstream job.

Also you will need to add a mechanism to update /etc/hosts file with external-name on nodes otherwise tests will fail after the deployment..